openapi.yml

```diff
@@ -146,19 +163,69 @@ components:
       format: FiscalCode
       x-import: italia-ts-commons/lib/strings
       example: SPNDNL80R13C555X
+    ArrayIndexUpTo4:
+      type: integer
+      minimum: -1
+      maximum: 5
+      exclusiveMaximum: true
+    ExistingArrayIndexUpTo4:
+      type: integer
+      minimum: 0
+      maximum: 5
+      exclusiveMaximum: true
+    NonEmptyString:
+      type: string
+      format: NonEmptyString
+      x-import: italia-ts-commons/lib/strings
     OrganizationFiscalCode:
       type: string
       description: Organization's fiscal code.
       format: OrganizationFiscalCode
       x-import: italia-ts-commons/lib/strings
       example: 01234567890
+    OrganizationScope:
+      type: string
+      enum:
+        - "LOCAL"
+        - "NATIONAL"
     UserRole:
       type: string
       enum:
         - "ORG_DELEGATE"
         - "ORG_MANAGER"
         - "DEVELOPER"
         - "ADMIN"
+    LegalRepresentative:
+      type: object
+      properties:
+        email:
+          $ref: "#/components/schemas/EmailAddress"
+        familyName:
+          $ref: "#/components/schemas/NonEmptyString"
+        firstName:
```

Comment:
The Italian standard is  `givenName` https://ontopia-lodview.prod.pdnd.italia.it/onto/CPV/givenName